go.vendor

```diff
@@ -0,0 +1 @@
+sigs.k8s.io/cluster-api
```

Comment:
What is this for?

Reply:
https://github.com/kubernetes-sigs/cluster-api/pull/1134